### ISSUE TITLE
Rehome FoodOn "agricultural food production" branch?

### ISSUE THREAD

**Issue by ddooley** [AgriculturalSemantics/agro#191]
Would AGRO be the suitable home for the "agricultural food production" branch currently in FoodOn?  

This may involve an expansion of scope for AGRO though, as it mentions **aquaculture**, **greenhousing**, and as well we have request for an "**animal husbandry production**" (process) and both "organic plant production" and "organic animal husbandry production" classes.  We are also interested in "certified organic plant production" and "certified organic animal husbandry production"

<img width="493" alt="image" src="https://github.com/AgriculturalSemantics/agro/assets/4000582/9cf2e5a6-fe72-44fc-b5a0-b4b662f013f8">

Would you like a pull request for this?

**Comment by KrishnaTO** [comment]
I would agree it does somewhat fit the 'Agronomy' part of this ontology, but the decision would be falling on whether @celineaubert or @marieALaporte agree?

**Comment by marieALaporte** [comment]
AGRO can be the home for the FoodOn "agricultural food production" branch. It would be great if you can create the PR. Let me know if you want to discuss over a call. 
AGRO started with crops, but we definitly want to expand beyond that, particularly in the context of the PTFI project.
